cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.curatoroperator.openshift.io/customresourcedefinition.yaml

```diff
@@ -0,0 +1,162 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: reports.curator.operatefirst.io
```

Comment:
just one comment , can you change the directory to this 
instead of `reports.curatoroperator.openshift.io`